files/create_clean_vo_data.py

```diff
@@ -0,0 +1,64 @@
+#!/bin/env python
+
+import json
+import requests
+from ruamel import yaml
+import jsbeautifier
+
+def get_data():
+    vars = yaml.safe_load(open('../defaults/main.yml'))
+    url = vars['lavoisier']['base_url'] + vars['lavoisier']['vo_id_card_endpoint']
+    try:
+        data = requests.get(url).json()
+    except UserWarning as e:
+        print e
```

Comment:
This won't work on python 3!

Reply:
I indeed was troubled by this too. Since Ansible is 2.7, I thought it might be better to stick to a consistent version. Given the upcoming deprecation of 2.7, I guess it won't hurt to have a python3-compatible one too. I'll open an issue for the next milestone.